index.bs

```diff
@@ -772,12 +766,12 @@ The <dfn method for=CookieStore>set(|options|)</dfn> method steps are:
 1. Run the following steps [=in parallel=]:
     1. Let |r| be the result of running [=set a cookie=] with
         |url|,
-        |options|' {{CookieStoreSetExtraOptions/name}} dictionary member,
-        |options|' {{CookieStoreSetExtraOptions/value}} dictionary member,
-        |options|' {{CookieStoreSetOptions/expires}} dictionary member,
-        |options|' {{CookieStoreSetOptions/domain}} dictionary member,
-        |options|' {{CookieStoreSetOptions/path}} dictionary member, and
-        |options|' {{CookieStoreSetOptions/sameSite}} dictionary member.
+        |options|' {{CookieInit/name}} dictionary member,
+        |options|' {{CookieInit/value}} dictionary member,
```

Comment:
`value` is not a required dictionary member now, but it's not an optional argument for the *set a cookie* algorithm, so we need to define what's passed here.

Can `value` be made required, or will that prevent something?

Reply:
Oh, good point.. I think this can be required. I will update.